packages/name/src/hasFamilyNameFirstNameOrdering.ts

```diff
@@ -0,0 +1,7 @@
+import {FAMILY_NAME_FIRST_NAME_ORDERING} from './constants';
+
+export function hasFamilyNameFirstNameOrdering(locale: string) {
```

Comment:
Aah, I just realized this should be named `hasFamilyNameGivenNameOrdering`, so that "given name" is aligned with the "given name" field we use in `formatName()`.

```suggestion
export function hasFamilyNameGivenNameOrdering(locale: string) {
```